src/include/cart/swim.h

```diff
@@ -59,9 +59,12 @@ typedef uint64_t swim_id_t;
 enum swim_member_status {
 	SWIM_MEMBER_ALIVE = 0,
 	SWIM_MEMBER_SUSPECT,
-	SWIM_MEMBER_DEAD
+	SWIM_MEMBER_DEAD,
+	SWIM_MEMBER_INACTIVE
 };
 
+#define SWIM_STATUS_CHARS "ASDI"
+
```

Comment:
Cool. Would it make sense to comment that the order of the chars must be the same with the order of the enum values in swim_member_status?

Reply:
ok